core/keyboard_nav/flyout_cursor.js

```diff
@@ -39,6 +39,33 @@ Blockly.FlyoutCursor = function() {
 };
 Blockly.utils.object.inherits(Blockly.FlyoutCursor, Blockly.Cursor);
 
+/**
+ * Handles the given action.
+ * This is only triggered when keyboard navigation is enabled.
+ * @param {!Blockly.Action} action The action to be handled.
+ * @return {boolean} True if the action has been handled, false otherwise.
+ * @override
+ */
+Blockly.FlyoutCursor.prototype.onBlocklyAction = function(action) {
+  // If we are on a field give it the option to handle the action
+  if (this.getCurNode() &&
```

Comment:
Does the flyout cursor ever navigate to a field?

Reply:
good point! fixed.